docs/local_setup_sans_docker.md

```diff
@@ -46,6 +77,13 @@ Now, you should be about ready to get started. This command in the top folder of
 bin/dev
 ```
 
+If you get an error related to `puma`, try uninstalling and re-installing `foreman`:
```

Comment:
Ah, okay. So I think it's not the installation of foreman that's a problem. Instead, I think it's what foreman does to the runtime environment. 

For context, this is the line in `puma.rb` that sets the Port of the server:
```
port = Integer(ENV['PORT'] || 3000)
```

Uninstalling and reinstalling `foreman` does not fix that port 5000 issue. But! If we remove foreman from the equation-- and basically run the contents of the `Procfile` that the `foreman` execution (in `bin/dev`) points to--directly: 
```
bundle exec puma -C config/puma.rb
```

I no longer have the issue. In fact, the server starts running on port 3000. To me, this indicates that there is something going on in the `foreman` execution environment that is setting the port to be 5000.

Reply:
I am not sure how we should handle this in our documentation. Do we even want to use `foreman` + `Procfile` if we are not planning on using heroku?